doc/source/whatsnew/v1.0.0.rst

```diff
@@ -207,6 +207,7 @@ Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 -
+- ``IndexError`` would not raise if all the values in a level of a MultiIndex were missing (:issue:`20519`)
```

Comment:
```suggestion
- Bug in :meth:`DataFrame.groupby` with multiple groups where an ``IndexError`` would be raised if any group contained all NA values (:issue:`20519`)
```

Reply:
Thank you for feedback. I  think about second test case and i also think duplicate part compare to first one. So i remove it.